src/uct/ib/rc/base/rc_ep.h

```diff
@@ -212,8 +218,8 @@ struct uct_rc_ep {
     ucs_arbiter_group_t arb_group;
     uct_rc_fc_t         fc;
     uint16_t            atomic_mr_offset;
+    uint16_t            flags;
```

Comment:
@hoopoepg is in necessary? can we use lower bits under AM_ID_BITS?

Reply:
those 3 bits are used for RC/DC flow control

Reply:
maybe reduce to 8 bits?